### DISCUSSION TITLE
working example using serverless framework and prisma

### DISCUSSION THREAD

**Discussion by omar-dulaimi** [MichalLytek/type-graphql#819] [Ideas]
Hello,

I'm looking for a recent working example for using typegrapqhl + prisma + serverless client. I'm running my code locally with serverless-offline, but so far it hasn't been an easy ride. Would appreciate a working example in the repo, as it would really help other people as well.

**Comment by MichalLytek** [comment]
What part of the stack is your problem? Can you run raw TypeGraphQL in serverless? Does Prisma connect do your database in serverless?

`typegraphql-prisma` is just a code generator, the same resolvers you would write by hand, so there's no special guide, tutorial or repo needed.

**Reply by omar-dulaimi** [reply]
Well I guess I asked at the wrong place. It should be at the other repo; TypeGraphQL. Or Am I not allowed to ask such a question?

**Reply by MichalLytek** [reply]
All the knowledge about TypeGraphQL in serverless is here:
https://github.com/MichalLytek/type-graphql/issues/96

>  I guess I asked at the wrong place

Your question about "using typegrapqhl + prisma" has mislead me so I thought your question is about `typegraphql-prisma`. Moving back to TypeGraphQL main repo 👀
